models/mongo/blocks.go

```diff
@@ -0,0 +1,60 @@
+package mongo
+
+import (
+	"context"
+	"notes-service/models"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type block struct {
+	ID      string `json:"id" bson:"_id,omitempty"`
+	NoteId  string `json:"noteId" bson:"noteId,omitempty"`
+	Type    uint32 `json:"type" bson:"type,omitempty"`
+	Content string `json:"content" bson:"content,omitempty"`
+}
```

Comment:
Why does this exist both in the models and the models.mongo package? Is there any reason for the duplication?